src/domain/errorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import styles from './errorPage.module.scss';
+import Text from '../../common/text/Text';
+import Icon from '../../common/icons/Icon';
+import InternalLink from '../../common/internalLink/InternalLink';
+
+const ErrorPage: React.FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.innerContainer}>
+        <Icon shape="IconExclamationCircle" size="large" />
+        <Text as="h1" className={styles.heading}>
+          {t('errorPage.heading')}
+        </Text>
+        <Text>{t('errorPage.text')}</Text>
+        <InternalLink to={'/'} className={styles.link}>
```

Comment:
Can you make this ExternalLink to force the user to reload the SPA after the error.

Reply:
Done 👍